ksql-engine/src/test/resources/query-validation-tests/more-complex-struct.json

```diff
@@ -0,0 +1,154 @@
+{
+  "comments": [
+    "You can specify multiple statements per test case, i.e., to set up the various streams needed",
```

Comment:
More useful comments would be useful i.e. what is this test file targeted at testing?

Reply:
Added more description.